cmd/containerd-shim-runhcs-v1/task_hcs.go

```diff
@@ -519,6 +519,26 @@ func (ht *hcsTask) GetExec(eid string) (shimExec, error) {
 	return raw.(shimExec), nil
 }
 
+func (ht *hcsTask) GetExecs() (_ []shimExec, err error) {
+	var execs []shimExec
+	ht.execs.Range(func(key, value interface{}) bool {
+		wt, loaded := value.(shimExec)
+		if !loaded {
```

Comment:
nit: `ok` feels more idiomatic than `loaded` here and above

Reply:
I'll change what I've added, we can swap the others if need be on another change